### DISCUSSION TITLE
Adding Background image From tailwind Config

### DISCUSSION THREAD

**Discussion by firestanf** [tailpress/tailpress#118] [Q&A] [answered]
Hello how to add background image from tailwind.config.js file 
im using this code to get the image from the theme folder

`module.exports = {`
`  theme: {`
`    extend: {`
`      backgroundImage: {`
`        'hero-pattern': "url('./img/HeroBackground.jpg')",`
`      }`
`    }`
`  }`
`}`

the problem is that when i compile with laravel mix it shows an error
for example

ERROR in ./resources/css/app.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
Error: Can't resolve './img/HeroBackground.jpg'

**Comment by jarednthomas** [comment, marked as answer]
I'm not using Laravel mix but I'm also not able to get it to work with a relative path. When done absolute, it works for me, but I'd rather not do it like this:

`'hero-pattern': "url('/wp-content/themes/<themeName>/img/HeroBackground.jpg')",
`
